helm-chart/values.yaml

```diff
@@ -27,3 +27,7 @@ auth:
   bearerTokenConfig:
     # A random token will be generated if this stays empty
     token:
+
+global:
+  postgresql:
+    existingSecret: "miniurl-postgresql"
```

Comment:
This will break if `{{ include "miniurl.storageSecretName" . }}` will change. You should unite those values into one (probably just replace all occurrences of `{{ include "miniurl.storageSecretName" . }}` with `{{ global.postgresql.existingSecret }}` or similar).